example/app/src/main/java/example/pilgrimsdk/kits/mparticle/com/pilgrimsdkmparticle/utils/Utils.kt

```diff
@@ -0,0 +1,13 @@
+package example.pilgrimsdk.kits.mparticle.com.pilgrimsdkmparticle.utils
+
+import android.Manifest
+import android.content.Context
+import android.content.pm.PackageManager
+import android.support.v4.content.ContextCompat
+
+fun isLocationPermissionGranted(ctx: Context): Boolean {
```

Comment:
If we are going to keep this external then you could just make it an extension function off the context so you don't have to pass anything in

```kotlin
fun Context.isLocationPermissionGranted(): Boolean
```